Makefile

```diff
@@ -67,6 +69,10 @@ ifeq ($(CONFIG_HDHOMERUN_STATIC),yes)
 CFLAGS  += -I${ROOTDIR}/libhdhomerun_static
 LDFLAGS += -L${ROOTDIR}/libhdhomerun_static/libhdhomerun \
            -Wl,-Bstatic -lhdhomerun -Wl,-Bdynamic
+
+ifneq (,$(filter gnueabihf,$(ARCH)))
+LDFLAGS += -ldl
```

Comment:
Ok, reject this PL. It still won't compile if I don't add -ldl (complains about libx264).